docs/src/content/en/reference/streaming/workflows/resume.mdx

```diff
@@ -0,0 +1,142 @@
+---
+title: "Reference: Run.resumeStream() | Workflows | Mastra Docs"
+description: Documentation for the `Run.resumeStream()` method in workflows, which enables real-time resumption and streaming of suspended workflow runs.
+---
+
+
+# Run.resumeStream() 
```

Comment:
Was this file supposed to be part of this? I think workflows.resumeStreamVNext() is a separate thing.